src/detect-pangram.js

```diff
@@ -15,7 +15,15 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  const alphabet = 'abcdefghijklmnopqrstuvwxyz';
+  const phraseLower = phrase.toLowerCase();
+  for (let i = 0; i < alphabet.length; i++) {
+    if (phraseLower.indexOf(alphabet.charAt(i)) === -1) {
```

Comment:
если нам не нужен индекс, то `includes`
```suggestion
    if (!phraseLower.includes) {
```

Reply:
понял, исправлю